tools/depends/native/meson/Makefile

```diff
@@ -33,3 +33,17 @@ clean:
 
 distclean::
 	rm -rf $(PLATFORM) .installed-$(PLATFORM)
+
+cross-file:
+	PREFIX="$(PREFIX)" \
+	NATIVEPREFIX="$(NATIVEPREFIX)" \
+	CC="$(CC)" \
+	CXX="$(CXX)" \
+	AR="$(AR)" \
+	STRIP="$(STRIP)" \
+	MESON_CPU="$(MESON_CPU)" \
+	CPU="$(CPU)" \
+	CFLAGS="$(CFLAGS)" \
+	CXXFLAGS="$(CXXFLAGS)" \
+	LDFLAGS="$(LDFLAGS)" \
```

Comment:
Using host (target) information in build (native) is wrong.

Reply:
our of curiosity, how would you approach this?

Split the meson cross file generation into a target dependency?

Reply:
Nevermind this. I've sent lrusak a bash script that generates the meson cross file purely in bash and can be executed as part of configure. No reliance on any native/target dependencies

Reply:
I consider the cross file as part of the meson build system so installing it with the native package is fine by me

Reply:
The more i think about this, the more i have to agree with Rechi.
Example use case:

I build for ios -> configure -> build native -> build target -> build kodi
I then want to build for a different platform (tvos)
all i should have to do is dist clean target -> configure -> build target -> build kodi.
native hasnt changed, there should be no need to rebuild native for a target change

With this currently, id have to go and rebuild native meson to regenerate the cross which is required due to flag/linker changes between platforms.

Either need to move the cross file generation to a target dependency, or my other option as a possibility. May be other choices as well, but it shouldnt stay in native

Reply:
sure